randovania/games/fusion/logic_database/Sector 2 TRO.txt

```diff
@@ -1360,6 +1379,7 @@ Extra - room_id: [43]
   > Door to Zoro Zig-Zag
       Any of the following:
           Space Jump
+          # https://youtu.be/8gjeZTK98_8
           Hi-Jump and Jump Extends (Advanced) and Wall Jump (Advanced)
```

Comment:
The video disagrees about needing HJ and jump extends.

The HJ path should probably just be HJ+WJ beginner. Maybe intermediate.